.github/workflows/release.yml

```diff
@@ -32,7 +32,9 @@ jobs:
 
       - name: Build artifacts
         run: |
-          pipx run build
+          pip install .
+          pdm install --plugins
```

Comment:
we don't need to change this for backend approach

Reply:
undone just now